### ISSUE TITLE
WFLW-V release

### ISSUE THREAD

**Issue by seuqaj114** [polo5/LDEQ_RwR#2]
Thanks for releasing the code! I managed to download the dataset as per your instructions, and happy to make it available here: https://drive.google.com/file/d/1N12tztn9skcwPwnncmRHX-T592O5nlaM/view?usp=sharing

**Comment by polo5** [comment]
Awesome thanks!

**Comment by gh786137611** [comment]
The WFLW-V download link seems to be broken, can you share again? Thanks.

> Thanks for releasing the code! I managed to download the dataset as per your instructions, and happy to make it available here: https://drive.google.com/file/d/1N12tztn9skcwPwnncmRHX-T592O5nlaM/view?usp=sharing

**Comment by polo5** [comment]
Of course, sorry about that. Please find the updated link here and let me know if something doesn't look right: https://drive.google.com/file/d/1YSJdgIb-vToJIAV04PGh_U7nX6dxVSjt/view?usp=share_link
